internal/driver/glfw/window_test.go

```diff
@@ -534,6 +534,22 @@ func TestWindow_HoverableOnDragging(t *testing.T) {
 	assert.NotNil(t, dh.popMouseOutEvent())
 }
 
+func TestWindow_Scrolled(t *testing.T) {
+	w := createWindow("Test").(*window)
+	o := &scrollable{Rectangle: canvas.NewRectangle(color.White)}
+	o.SetMinSize(fyne.NewSize(100, 100))
+	w.SetContent(o)
+
+	w.mousePos = fyne.NewPos(50, 60)
+	w.mouseScrolled(w.viewport, 10, 10)
+	w.waitForEvents()
+
+	if e, _ := o.popScrollEvent().(*fyne.PointEvent); assert.NotNil(t, e, "scroll event") {
```

Comment:
Should be `fyne.ScrollEvent` instead of `fyne.PointEvent`. Not sure, how the tests passed before 🤔

Reply:
Indeed it should have been. However it works just fine because ScrollEvent is a PointEvent as well